Extension/src/LanguageServer/settings.ts

```diff
@@ -461,6 +463,33 @@ export class CppSettings extends Settings {
         return super.Section.get<boolean>("inlayHints.referenceOperator.showSpace") === true;
     }
 
+    public get otfDocsEnabled(): PromiseLike<boolean> {
+        // Check if the user has access to copilot.
+        return vscode.lm.selectChatModels({vendor: "copilot"}).then((models) => {
+            // Check if the setting is explicitly set to enabled or disabled.
+            const setting = super.Section.get<string>("onTheFlyDocsEnabled");
```

Comment:
Probably doesn't matter, but we might want to check this first so we can bail early if it's `disabled` and avoid interacting with the language model API at all.